{
  throw new RuntimeException("Intentional exception from 'execute'");
}
